3rdparty/libtengine/tengine.cmake

```diff
@@ -33,8 +32,9 @@ IF(EXISTS "${OCV_TENGINE_SOURCE_PATH}")
 	SET(BUILD_TENGINE ON)
 ELSE()
 	SET(OCV_TENGINE_FILENAME "${TENGINE_COMMIT_VERSION}.zip")#name2
-	SET(OCV_TENGINE_URL "https://github.com/OAID/Tengine/archive/") #url2
-	SET(tengine_md5sum f51ca8f3963faeeff3f019a6f6edc206) #md5sum2
+	#SET(OCV_TENGINE_URL "https://github.com/OAID/Tengine/archive/") #url2
+	SET(OCV_TENGINE_URL "https://github.com/liqi-c/Tengine/archive/") #url2.  Tmp url
```

Comment:
You don't need that for testing. Upstream URL should work fine with commits from fork.

Reply:
done.